4.6.2/Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:4.6.2
 
-RUN powershell -Command Add-WindowsFeature NET-Framework-45-ASPNET; \
-    powershell -Command Add-WindowsFeature Web-Asp-Net45; \
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
This surprised me as well; this is a breaking change for an existing tag. If a PowerShell version was desirable for an existing tag, it would've been better to add a new image version and tag (e.g. `microsoft/aspnet:4.6.2-powershell`)

I'm triaging an issue reported for Docker; https://github.com/moby/moby/issues/34013 and _suspect_ this may be the cause